src/layers/models/layersManager.ts

```diff
@@ -258,7 +256,7 @@ class LayersManager {
   }
 
   private addNewCache(jsonDocument: IMapProxyJsonDocument, layerRequest: ILayerPostRequest): void {
-    if (this.redisConfig.enabled) {
+    if (layerRequest.isRedis) {
```

Comment:
where is the changes in OpenAPI?
Whydo you pass the isRedis in update layer request if it's isn't in use?